### ISSUE TITLE
Not running on custom hardware

### ISSUE THREAD

**Issue by jdwEmSys** [meta-homeassistant/meta-homeassistant#71]
Hello

I’m busy with building Home-Assistant on my own hardware (not RPI) now I’m stuck.
I cloned the git of home-assistant and add IMAGE_INSTALL += " python3-homeassistant" to my local config file. 
Add the BBlayer and stuff.. 
After building and booting I got a web GUI, when I try to login after configuration. I got error, Cannot save during onboarding initial setup. 
How can I solve this problem?

**Comment by bachp** [comment]
If I understand correctly you are able to access the home assistant web UI but you can't sucessfully configure it?

Are you using a read only rootfs?

**Comment by jdwEmSys** [comment]
Yes, you understand right.
No i can access and create files on my system.

**Comment by bachp** [comment]
The version of home assistant in kirkstone is over a year old and not well tested. Could you try with the main branch? It is currently targeting Mickledore.

**Comment by Gerry546** [comment]
Correct, and on top of that if might be usefull to also check if there are any logs printed in the journal that might be helpfull. It might be of a dependency which might not be included in the default configuration of python3 when building it via yocto.
